utils/src/main/java/com/cloud/maint/Version.java

```diff
@@ -19,59 +19,82 @@
 
 package com.cloud.maint;
 
+import java.util.regex.Pattern;
+
+import org.apache.commons.lang.StringUtils;
+
 public class Version {
     /**
      * Compares two version strings and see which one is higher version.
-     * @param ver1
-     * @param ver2
-     * @return positive if ver1 is higher.  negative if ver1 is lower; zero if the same.
+     *
+     * @param version1
+     * @param version2
+     *
+     * @return positive if version1 is higher.  negative if version2 is lower; zero if the same.
```

Comment:
this comment is wrong:
version1 higher
version2 lower

Reply:
Nice finding!